companion/manifest.json

```diff
@@ -0,0 +1,26 @@
+{
+	"id": "monteiro-pptvideoosc",
+	"name": "monteiro-pptvideoosc",
+	"shortname": "monteiro-pptvideoosc",
+	"description": "Perform operations on separate machines.",
```

Comment:
Please use a more precise and descriptive description. "operations" can be anything, "machines" can be anything